tests/jepsen.clickhouse-keeper/resources/keeper_config.xml

```diff
@@ -21,7 +21,8 @@
             <heart_beat_interval_ms>1000</heart_beat_interval_ms>
             <election_timeout_lower_bound_ms>2000</election_timeout_lower_bound_ms>
             <election_timeout_upper_bound_ms>4000</election_timeout_upper_bound_ms>
-            <quorum_reads>{quorum_reads}</quorum_reads>
+            <quorum_reads>0</quorum_reads>
+            <read_mode>fastlinear</read_mode>
```

Comment:
I would return this to the old state, and see what we can do to test both quorum and fastlinear with jepsen in different PR.